### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by jbust23**

### CODE REVIEW

**Review by Mejiro-McQueen — APPROVED** [review]

File: ait/gui/static/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ait-gui",
-  "version": "2.3.1",
+  "version": "2.3.2-rc1",
```
Comment (cSchubes) [inline ait/gui/static/package.json]: do we care about this version at all? should it match the version we set in `setup.py`? I think this file is actually overwritten by https://github.jpl.nasa.gov/SunRISE-Ops/SunRISE-AIT/blob/master/gui/static/package.json in SunRISE-AIT. But, question still remains for this repo

File: doc/source/conf.py [hunk 2]
```diff
@@ -72,9 +72,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'2.3.1'
+version = u'2.3.2-rc1'
```
Comment (cSchubes) [inline doc/source/conf.py:75]: same question for this line and line 77 as for the package.json

**Review by cSchubes — COMMENTED** [review]
If we don't care about these versions, good to merge. Otherwise, should change them to match setup.py. It seems it doesn't matter from a functionality perspective

**Review by cSchubes — APPROVED** [review]
approving so merge can happen despite questions about version numbers

File: doc/source/conf.py (hunk 2 above)
Comment (jbust23) [reply doc/source/conf.py:75]: I fixed the packjage.json but left the docs untouched. Will either fix those later

**Pull request merged**